### ISSUE TITLE
KLayout integration installation issue

### ISSUE THREAD

**Issue by GuanhuiWang** [lukasc-ubc/SiEPIC-Tools#60]
Hi Lukas,

Following is a problem reported by a customer when try to install the integration:

"I first uninstalled the Klayout then installed it again and follow the way as told in GitHub

lukasc-ubc/SiEPIC-Tools

SiEPIC-Tools - Package for KLayout to add integrated optics / silicon photonics functionality (waveguides, netlist extraction, circuit simulations, etc)
I went to tools --> Manage Packages, --> installed the packages siepic_tools_0.3.18 and also siepic_ebeam_pdk 0.3.5

I have already installed Python and Numpy library on my computer.

Do I need to install the windows_python_package_for_kKlapout also ??
I run all the command line prompt which starts with mlink

I get this error when I first try to run SiEPIC 0.3.16 --> SImulation --> Setup Lumerical Interconnect Compact Model Library

![beec0790bf8b9133c96adac1c5654636e88a73b6](https://user-images.githubusercontent.com/22895092/35541236-a8f5fe6a-050e-11e8-8c3d-633a3f3c9412.png)

I have very limited knowledge on this. Please help to comment on this problem. "

Thank you,
Guanhui

**Comment by lukasc-ubc** [comment]
Yes, windows_python_package_for_KLayout is required for windows users.  It isn’t easy to get KLayout to use separately a installed Python installation (it would need to be exactly the same version as the one which the software was compiled with).

The mklink commands are only for developers.  For end users, the only requirement is KLayout 0.25, and SiEPIC-tools 0.3.22, and the windows packages.

**Comment by GuanhuiWang** [comment]
Thank you Lukas. I will let the customer know.
